@xen-orchestra/audit-core/src/index.js

```diff
@@ -63,10 +66,12 @@ export class AuditCore {
   }
 
   @defer
-  async add($defer, subject, event, data) {
+  async add($defer, subject, event, data, force = false) {
```

Comment:
Don't do this, split the method in a public method with the lock and a private method which does not lock and maybe make it explicit in the name (for example `_addUnsafe`).